input/otlp/logs.go

```diff
@@ -84,6 +85,13 @@ func (c *Consumer) convertInstrumentationLibraryLogs(
 	}
 }
 
+func isError(record plog.LogRecord) bool {
```

Comment:
```suggestion
func isExceptionLogRecord(record plog.LogRecord) bool {
```

Let's make sure the function name makes sense at the package level, not just within the file. Also, I think it's important that we distinguish between "error logs" and "exception logs". "Error" might just imply that it's a log message with error log level, which we don't automatically turn into error events.

Reply:
this function has been removed!